src/clj/puppetlabs/services/jruby/jruby_puppet_schemas.clj

```diff
@@ -137,7 +138,8 @@
    :profiler              (schema/maybe PuppetProfiler)
    :pool-agent            JRubyPoolAgent
    :flush-instance-agent  JRubyPoolAgent
-   :pool-state            PoolStateContainer})
+   :pool-state            PoolStateContainer
+   :shared-terminus-state (schema/maybe Map)})
```

Comment:
@cprice404 Thinking ahead to the work that we're planning on undertaking soon around the JRuby pool / library refactor, I wonder if you think this is the best place to store this kind of state.  Maybe it would be better to have this as a top-level piece of data in whatever consuming `puppet` Trapperkeeper service rather than putting it right into the pool structure?  Dunno.  Suppose we can move this around later on as we figure out exactly what the new APIs will look like so long as we'd preserve PuppetDB's ability to share state across JRuby instances using the same Ruby-level API.